src/Vendor/Order/Order.php

```diff
@@ -108,6 +111,7 @@ public function __construct(array $data)
         }, $data['shipment_ids']);
         $this->shippingCost = $data['shipping_cost'];
         $this->notes = $data['notes'];
+        $this->vendorComment = $data['details'] ?? '';
```

Comment:
```suggestion
        $this->vendorComment = $data['details'] ?? null;
```

Perso je trouve ça plus utile d'avoir `null` si la valeur n'est pas setté plutôt que de croire que le commentaire est vide